pylint_pytest/checkers/types.py

```diff
@@ -1,10 +1,12 @@
+from __future__ import annotations
+
 import sys
 from pprint import pprint
-from typing import Any, Dict, List
+from typing import Any
 
 from _pytest.fixtures import FixtureDef
 
-FixtureDict = Dict[str, List[FixtureDef[Any]]]
```

Comment:
I think you are not able to do that btw 😓 

_I don't understand why, but `ruff` / `pyupgrade` know the difference_

Do you want to revert that locally, and see what `pre-commit` offers? (pref. locally, but CI will catch it anyway)